kytos/core/controller.py

```diff
@@ -25,7 +25,8 @@
 from kytos.core.api_server import APIServer
 from kytos.core.buffers import KytosBuffers
 from kytos.core.events import KytosEvent
-from kytos.core.helpers import now, start_logger
+from kytos.core.helpers import now
+from kytos.core.log_manager import LogManager
```

Comment:
Thinking about our discussions to reduce imports sizes, we could do:
```
from kytos.core import LogManager
```
can't we?